app/src/main/java/org/fossasia/phimpme/uploadhistory/UploadHistory.java

```diff
@@ -165,11 +173,12 @@ public void execute(Realm realm) {
        // ArrayList<UploadHistoryRealmModel> ki = new ArrayList<>();
        // String s = preferenceUtil.getString("upload_view_choice", "Last first");
         uploadResults = new ArrayList<>();
-        if(displaychoice.equals(getString(R.string.last_first))){
+        if(displaychoice.equals(getString(R.string.last_first)) && uploadHistoryRealmModelRealmQuery.count() != 0){
             for(int i = 0; i < uploadHistoryRealmModelRealmQuery.findAll().size(); i++){
+                Log.i("lalalal", uploadHistoryRealmModelRealmQuery.findAll().get(i).getPathname());
```

Comment:
It would be awesome if you could remove this

Reply:
@abishekvashok Sure, sorry I forgot to delete it while pushing the commit.